spec/configuration_spec.lua

```diff
@@ -125,6 +125,39 @@ describe('Configuration object', function()
       assert.same(services, filter_services(services, { '42' }))
       assert.same({}, filter_services(services, { '21' }))
     end)
+
+    describe("with service filter", function()
```

Comment:
We should probably add a test that combines `APICAST_SERVICES_FILTER` and `APICAST_SERVICES_LIST`.